scripts/helpers.py

```diff
@@ -746,3 +746,16 @@ def read_csv_nafix(file, **kwargs):
         return pd.read_csv(file, **kwargs)
     else:
         return pd.DataFrame()
+
+
+def safe_divide(numerator, denominator):
```

Comment:
cool!
usually, I like to specify the default value as optional parameter, e.g. 
```suggestion
def safe_divide(numerator, denominator, default_value=np.nan):
```

This usually generalizes quite well.
I see you used "logging": is it defined? I fear it may throw an error, have you tested it?

Reply:
@davide-f Agree :) tested the logging --> works